app/components/layout/FindHeader.jsx

```diff
@@ -22,8 +22,17 @@ class FindHeader extends React.Component {
   render() {
     return (
       <header className="hero header-large" role="banner">
-        <h1>Hello, what can we help you find?</h1>
-        <h3>Search 1,138 housing and homelessness related services in San Francisco</h3>
+        <div className="header-container">
+          <div className="header-text">
+            <p>Welcome to the SF Service Guide</p>
```

Comment:
I have the same question on https://github.com/ShelterTechSF/askdarcel-web/pull/609#pullrequestreview-188890498, but I think we may want to make "SF Service Guide" parameterizable based on the URL, since I *think* we only whitelabel the app for the city when it's at a certain domain name, and we otherwise still want the app to be called Ask Darcel.